### PR TITLE
Updating cluster backup and restore

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/stolostron/backlog/issues/24514

### CODE REVIEW

File: backup_restore/backup_and_restore.adoc [hunk 1]
```diff
@@ -161,7 +161,7 @@ Refer to the link:https://github.com/openshift/oadp-operator/blob/master/docs/co
...
 
-You can use `serverSideEncryption` and `kmsKeyId` parameters to enable encryption for the backups stored in Amazon S3. For more details, see the link:https://github.com/vmware-tanzu/velero-plugin-for-aws/blob/main/backupstoragelocation.md[Backup Storage Location YAML]. The following sample specifies an AWS KMS key ID when setting up the `DataProtectionApplication` resource:
+It is strongly recommended to encrypt the data at the destination using the available backup storage server-side encryption. The backup contains resources, such as credentials and configuration files that need to be encrypted when stored outside of the hub cluster.You can use `serverSideEncryption` and `kmsKeyId` parameters to enable encryption for the backups stored in Amazon S3. For more details, see the link:https://github.com/vmware-tanzu/velero-plugin-for-aws/blob/main/backupstoragelocation.md[Backup Storage Location YAML]. The following sample specifies an AWS KMS key ID when setting up the `DataProtectionApplication` resource:
```
Comment (sahare) [inline backup_restore/backup_and_restore.adoc]: Please add a space in between these two sentences: "hub cluster.You can use"

**Review by sahare — APPROVED** [review]
Looks good, Thanks, just added a small comment.

File: backup_restore/backup_and_restore.adoc (hunk 1 above)
Comment (swopebe) [reply backup_restore/backup_and_restore.adoc]: Special tag here for **Best practice:**

Comment (swopebe) [reply backup_restore/backup_and_restore.adoc]: and as you guys saw, I moved the files lol, will create the book soon so you can see it.

**Review by swopebe — APPROVED** [review]

**Pull request merged**